src/irmin/store_properties_intf.ml

```diff
@@ -69,4 +87,9 @@ module type Sigs = sig
     include Clearable
     (** @inline *)
   end
+
+  module type Checkable = sig
```

Comment:
I presume this one can't be merged with `Closeable`?

Reply:
Yes this functor just adds a check on top for backends which doesn't support double-closing properly. Maybe I should just name this `Double_closeable`? That sounds a bit odd though...